caluma/caluma_workflow/serializers.py

```diff
@@ -467,17 +467,6 @@ class SaveWorkItemSerializer(SendEventSerializerMixin, ContextModelSerializer):
         help_text=models.WorkItem._meta.get_field("description").help_text,
     )
 
-    @transaction.atomic
-    def update(self, instance, validated_data):
-        self.send_event(
```

Comment:
The SendEventSerializerMixin might no longer be necessary either

Reply:
Yes, fixed!